docker/Dockerfile.cross_compile_aarch64

```diff
@@ -0,0 +1,62 @@
+FROM centos:7.6.1810
+
+ARG gcc_version=4.9-2016.02
+ARG openssl_version=1_1_1d
+ARG apr_version=1.6.5
+
+ENV GCC_VERSION $gcc_version
+ENV OPENSSL_VERSION $openssl_version
+ENV APR_VERSION $apr_version
+
+# Install requirements
+RUN  set -x && \
+  yum -y install epel-release && \
+  yum -y install wget tar git make autoconf automake libtool openssl-devel ninja-build gcc-c++
+
+# Install Java and Golang
+RUN yum install -y java-1.8.0-openjdk-devel golang
+ENV JAVA_HOME="/usr/lib/jvm/java-1.8.0-openjdk/"
+
+# Install aarch64 gcc 4.9 toolchain
+RUN set -x && \
+  wget https://releases.linaro.org/components/toolchain/binaries/$GCC_VERSION/aarch64-linux-gnu/gcc-linaro-$GCC_VERSION-x86_64_aarch64-linux-gnu.tar.xz && \
+  tar xvf gcc-linaro-$GCC_VERSION-x86_64_aarch64-linux-gnu.tar.xz
+ENV PATH="/gcc-linaro-$GCC_VERSION-x86_64_aarch64-linux-gnu/bin:${PATH}"
+
+# Install CMake
+RUN yum install -y cmake3 && ln -s /usr/bin/cmake3 /usr/bin/cmake
+
+# Cross compile Apache Apr for aarch64 - static
+RUN set -x && \
+  wget https://downloads.apache.org//apr/apr-$APR_VERSION.tar.gz && \
+  tar xvf apr-$APR_VERSION.tar.gz && \
+  pushd apr-$APR_VERSION && \
+  CFLAGS='-O3 -fno-omit-frame-pointer -fPIC' ./configure --disable-shared --prefix=/opt/apr-$APR_VERSION-static --host=aarch64-linux-gnu ac_cv_file__dev_zero=yes ac_cv_func_setpgrp_void=yes apr_cv_tcp_nodelay_with_cork=yes ac_cv_sizeof_struct_iovec=8 && \
+  make || true && \
+  pushd tools && \
+  gcc -Wall -O2 -DCROSS_COMPILE gen_test_char.c -s -o gen_test_char && \
+  popd && \
+  make && make install && \
+  popd
+
+# Cross compile Apache Apr for aarch64 - share
+RUN set -x && \
+  wget https://downloads.apache.org//apr/apr-$APR_VERSION.tar.gz && \
+  tar xvf apr-$APR_VERSION.tar.gz && \
+  pushd apr-$APR_VERSION && \
+  CFLAGS='-O3 -fno-omit-frame-pointer -fPIC' ./configure --prefix=/opt/apr-$APR_VERSION-share --host=aarch64-linux-gnu ac_cv_file__dev_zero=yes ac_cv_func_setpgrp_void=yes apr_cv_tcp_nodelay_with_cork=yes ac_cv_sizeof_struct_iovec=8 && \
```

Comment:
@wangxiyuan hm... wouldn't this basically have the dynamic linked tcnative version only work if the user has apr installed in the same path when trying to use `netty-tcnative` ?